packages/terra-collapsible-menu-view/src/CollapsibleMenuViewDivider.jsx

```diff
@@ -2,22 +2,45 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Menu from 'terra-menu';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
 import ThemeContext from 'terra-theme-context';
 import styles from './CollapsibleMenuView.module.scss';
 
 const cx = classNames.bind(styles);
 
+const propTypes = {
+  /**
+   * @private
+   * Uses a ">" icon instead of a line.
+   */
+  useChevronRightIcon: PropTypes.bool,
+};
+
 const contextTypes = {
   isCollapsibleMenuItem: PropTypes.bool,
 };
 
-const CollapsibleMenuViewDivider = (props, { isCollapsibleMenuItem }) => {
+const CollapsibleMenuViewDivider = ({ useChevronRightIcon }, { isCollapsibleMenuItem }) => {
```

Comment:
Should we change this from props?  What if we want to add additional props in the future?  Do we want to have a default prop value for this?

Reply:
Yes, you can destructure.  i get how it is working.  I guess my concern is for readability.  You could just as easily destructure from props.